.github/workflows/ios-debug-run.yaml

```diff
@@ -0,0 +1,49 @@
+name: ios debug run
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  ios_debug_run:
+    # Creates a build matrix for the jobs. You can define different
+    # variations of an environment to run each job in.
+    strategy:
+      # A set of different configurations of the virtual environment.
+      matrix:
+        device:
+        - "iPhone 8 (13.3)"
```

Comment:
Will this debug only on IOS version 13.3?

Reply:
No this is just the iOS version of the emulator that is available, which should be backwards compatible with whatever version of the app we have. I have verified this by building 13.2 in xcode and running it on this simulator.

Reply:
Alright cool sounds good, everything else looks good to me